src/pages/[guild]/index.tsx

```diff
@@ -224,6 +226,9 @@ const GuildPage = (): JSX.Element => {
         backgroundImage={localBackgroundImage}
         action={isAdmin && <DynamicEditGuildButton />}
         backButton={{ href: "/explorer", text: "Go back to explorer" }}
+        titlePostfix={
+          tags && tags.includes("VERIFIED") && <VerifiedIcon iconSize={6} mt={-2} />
```

Comment:
I think we could use `tags?.includes("VERIFIED")` here, that would be a bit shorter